app/javascript/mastodon/features/status/index.jsx

```diff
@@ -184,6 +185,14 @@ const titleFromStatus = (intl, status) => {
   return text ? `${user}: "${truncate(text, 30)}"` : intl.formatMessage(messages.statusTitleWithAttachments, { user, attachmentCount });
 };
 
+const RemoteHint = ({ url }) => (
+  <TimelineHint url={url} resource={<FormattedMessage id='timeline_hint.resources.replies' defaultMessage='Some replies' />} />
```

Comment:
I'd suggest doing this inline given it's a single line component that's not reused elsewhere. React should be smart enough to not re-render this needlessly.

Reply:
Done